config/management/genesis/README.md

```diff
@@ -121,70 +129,58 @@ where each field maps to a role as described in this document.
+    --config config_file.yaml \
+    --operator-name OPERATOR_NAME
 ```
 
 ### Validator Operators
 
 * Each Validator Operator member will upload their key to GitHub:
 ```
-cargo run -p libra-genesis-tool -- \
+cargo run -p libra-genesis-tool --
     operator-key \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
+    --config config_file.yaml
 ```
-* For each validator managed by an operator, the operator will upload a signed
-  validator-config. The owner corresponds to the name of the validator owner (as
-  registered in the shared Github). The namespace in GitHub correlates to the
-  operator namespace:
+* For each validator managed by an operator, the operator will upload a signed validator-config. The owner corresponds to the name of the OW (as registered in the shared Github). The namespace in GitHub correlates to the operator namespace:
 ```
-cargo run -p libra-genesis-tool -- \
+cargo run -p libra-genesis-tool --
     validator-config \
+    --config config_file.yaml \
     --owner-name OWNER_NAME \
     --validator-address '/dns/DNS/tcp/PORT' \
     --fullnode-address '/dns/DNS/tcp/PORT' \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
 ```
-* Upon receiving signal from the association, validator operators can now build
-  genesis, this requires no namespace:
+* Upon receiving signal from the association, OPs can now build genesis:
 ```
 cargo run -p libra-genesis-tool -- \
     genesis \
+    --config config_file.yaml \
     --path PATH_TO_GENESIS \
-    --backend 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN'
 ```
-* Upon receiving signal from the association, validator operators can now build
-  a genesis waypoint, this requires no namespace.  In this command, the remote
-  store is the destination where the waypoint will be saved. It is derived from
-  data in the local backend:
+* Similarly, the association should publish a genesis waypoint, and the OP should insert it into their storage (using the management tool):
 ```
-cargo run -p libra-genesis-tool -- \
-    create-and-insert-waypoint \
-    --local 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN' \
-    --remote 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN'
+cargo run -p libra-genesist-tool -- \
+    insert-waypoint \
+    --config config_file.yaml \
+    --waypoint WAYPOINT
 ```
 * Perform a verify that ensures the local store maps to Genesis and Genesis maps
-  to the waypoint. (TBD)
+  to the waypoint:
+```
+cargo run -p libra-genesis-tool -- \
+    verify \
+    --config config_file.yaml \
+    --genesis_path PATH_TO_GENESIS
```

Comment:
One thing that the operators pointed out earlier, is with runbooks to use variables e.g. `$PATH_TO_GENESIS` so you can just set the variable and copy and paste.  But, I think that's really personal preference at this point.

Reply:
interesting, I wasn't present for that, I'm not used to see that model, but I'll slap another commit on top since it doesn't really change the meaning!